### ISSUE TITLE
Include full checkpoint

### ISSUE THREAD

**Issue by mhamilton723** [facebookresearch/msn#12]
Hey thanks so much for making this code public. I'm trying to use this in a project and would love to be able to continue the training process. Could you please upload a checkpoint with all of the optimizer state? Thanks so much for your help!

**Comment by Aaditya-Singh** [comment]
It'd also be great if the finetuned checkpoint can be released. I tried to finetune MSN ViTB-16 with [MAE's finetuning configs](https://github.com/facebookresearch/mae/blob/main/FINETUNE.md#fine-tuning) (as mentioned [here](https://github.com/facebookresearch/msn#imagenet-1k-fine-tuning-evaluation)) but it only reaches to 82.2% accuracy on IN1k validation set, as opposed to 83.4% mentioned in Table 4 of the paper, and also does worse on the distribution shifts.
